Sakura/Modules/payments.py

```
@@ -0,0 +1,217 @@
+
+        if message.chat.type == ChatType.PRIVATE:
+            await send_effect(message.chat.id, buyers_text)
+        else:
+            await message.reply_text(buyers_text, parse_mode=ParseMode.HTML, link_preview_options=LinkPreviewOptions(is_disabled=True))
+
+    except Exception as e:
+        user_info = fetch_user(message)
+        log_action("ERROR", f"❌ Error in buyers command: {e}", user_info)
+        await message.reply_text(get_error())
+
+async def send_invoice(client: Client, chat_id: int, user_info: dict, amount: int, message_effect_id: str = None):
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)